### ISSUE TITLE
dehorizontalized beam

### ISSUE THREAD

**Issue by craigsapp** [rism-ch/verovio#213]
Here is a strange condition for beams that I noticed:

![screen shot 2016-06-25 at 5 14 11 pm](https://cloud.githubusercontent.com/assets/3487289/16359798/57ca949c-3af8-11e6-9790-4c472fc0ff95.png)

The beam in both cases should be horizontal, but the second one has an angle.  Perhaps it is the result of a bug?

MEI data:

``` xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="http://music-encoding.org/schema/3.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="http://music-encoding.org/schema/3.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="3.0.0">
    <meiHead>
        <fileDesc>
            <titleStmt>
                <title />
            </titleStmt>
            <pubStmt>
                <date>2016-06-25 17:14:16</date>
            </pubStmt>
        </fileDesc>
        <encodingDesc>
            <projectDesc>
                <p>Encoded with Verovio version 0.9.12-dev-b005b99-dirty</p>
            </projectDesc>
        </encodingDesc>
    </meiHead>
    <music>
        <body>
            <mdiv>
                <score>
                    <scoreDef xml:id="scoredef-000000170365610">
                        <staffGrp xml:id="m-000000085556223">
                            <staffDef xml:id="staffdef-000000203150059" clef.shape="G" clef.line="2" n="1" lines="5" />
                        </staffGrp>
                    </scoreDef>
                    <section>
                        <measure xml:id="measure-000000058797970" n="0">
                            <staff xml:id="staff-000000018741566" n="1">
                                <!-- kern: **kern  *clefG2  8c 8eL  8eJ  = -->
                                <layer xml:id="layer-000000168412238" n="1">
                                    <beam xml:id="beam-000000121039000">
                                        <chord xml:id="chord-000000205974326" dur="8">
                                            <note xml:id="note-000000066866521" oct="4" pname="c" accid.ges="n" />
                                            <note xml:id="note-000000047434375" oct="4" pname="e" accid.ges="n" />
                                        </chord>
                                        <note xml:id="note-000000083617581" dur="8" oct="4" pname="e" accid.ges="n" />
                                    </beam>
                                </layer>
                            </staff>
                        </measure>
                        <measure xml:id="measure-000000047391993" n="32758" right="invis">
                            <staff xml:id="staff-000000184827178" n="1">
                                <!-- kern: =  8c 8eL  8e  8c  8eJ  =- -->
                                <layer xml:id="layer-000000055290302" n="1">
                                    <beam xml:id="beam-000000047941741">
                                        <chord xml:id="chord-000000020981674" dur="8">
                                            <note xml:id="note-000000022191845" oct="4" pname="c" accid.ges="n" />
                                            <note xml:id="note-000000175186199" oct="4" pname="e" accid.ges="n" />
                                        </chord>
                                        <note xml:id="note-000000154371598" dur="8" oct="4" pname="e" accid.ges="n" />
                                        <note xml:id="note-000000148455326" dur="8" oct="4" pname="c" accid.ges="n" />
                                        <note xml:id="note-000000142162997" dur="8" oct="4" pname="e" accid.ges="n" />
                                    </beam>
                                </layer>
                            </staff>
                        </measure>
                    </section>
                </score>
            </mdiv>
        </body>
    </music>
</mei>
```

**Comment by lpugin** [comment]
Fixed in 2.7.0

**Comment by craigsapp** [comment]
Looks great:

<img width="436" alt="Screen Shot 2020-03-22 at 10 32 42" src="https://user-images.githubusercontent.com/3487289/77255978-8c815800-6c28-11ea-8b5c-1bdb92d936e5.png">

And I like the placement of the beam better in the new version.
